src/components/picker/helpers/usePickerMode.ts

```diff
@@ -0,0 +1,47 @@
+import {ExpandableOverlayProps} from 'src/incubator';
+import {PickerProps, PickerModeTypes} from '../index';
+import {CustomPickerProps} from '../types';
+
+type PickerType = {
+  dialog?: boolean;
+  wheelPicker?: boolean;
+  modal?: boolean;
+  custom?: boolean;
+};
+
+const usePickerMode = (props: PickerProps) => {
```

Comment:
Why is it named "usePickerMode" and not "usePickerType"?

Reply:
renamed.